src/pages/RepoPage/CommitsTab/CommitsTableTeam/createCommitsTableTeamData.tsx

```diff
@@ -0,0 +1,61 @@
+import isArray from 'lodash/isArray'
+import isEmpty from 'lodash/isEmpty'
+
+import { Commit } from 'services/commits/useCommitsTeam'
+import TotalsNumber from 'ui/TotalsNumber'
+
+import CIStatus from '../shared/CIStatus'
+import Title from '../shared/Title'
+
+export const createCommitsTableTeamData = ({
+  pages,
+}: {
+  pages?: Array<{ commits: Array<Commit | null> }>
+}) => {
+  let commits: Array<Commit | null> = []
+  if (isArray(pages) && !isEmpty(pages)) {
```

Comment:
I think you can either make pages mandatory or [return early](https://medium.com/swlh/return-early-pattern-3d18a41bba8) if pages is empty. 

Something like 

`  if (!isArray(pages)) return [];
`
`  const commits: Array<Commit | null> = pages.map((page) => page?.commits).flat();
`
`  if (isEmpty(commits)) return [];
`

Reply:
Done